src/mcmcchains-connect.jl

```diff
@@ -0,0 +1,35 @@
+import .MCMCChains: Chains
+
+# A basic chains constructor that works with the Transition struct we defined.
+function AbstractMCMC.bundle_samples(
+    ts::Vector{<:Transition},
+    model::AbstractMCMC.AbstractModel,
+    sampler::AbstractMCMC.AbstractSampler,
+    state,
+    chain_type::Type{Chains};
+    discard_initial=0,
+    thinning=1,
+    param_names=missing,
+    bijector=identity,
+    kwargs...
+)
+    # Turn all the transitions into a vector-of-vectors.
+    t = ts[1]
+    tstat = merge((; lp = t.z.ℓπ.value), stat(t))
+    tstat_names = collect(keys(tstat))
+    vals = [vcat(bijector(t.z.θ), collect(tstat)) for t in ts]
```

Comment:
Currently this adds the statistics from the first transition to the values of every transition, but most of the statistics change with every transition. Something like this should be right:
```suggestion
    vals = [vcat(bijector(t.z.θ), t.z.ℓπ.value, collect(values(stat(t)))) for t in ts]
```